infra/build/postgres/Dockerfile

```diff
@@ -23,16 +23,22 @@ RUN set -eux; \
          ;; \
     esac;
 
-RUN apt update && apt install -y curl
+RUN apt update && apt install curl -y
 
 # Install precompiled pg_graphql extensions
 RUN curl -L "https://github.com/supabase/pg_graphql/releases/download/v${PG_GRAPHQL_VERSION}/pg_graphql-v${PG_GRAPHQL_VERSION}-pg${PG_MAIN_VERSION}-${TARGETARCH}-linux-gnu.deb" -o pg_graphql.deb
 RUN dpkg --install pg_graphql.deb
-RUN cp /usr/share/postgresql/15/extension/pg_graphql* /opt/bitnami/postgresql/share/extension/
-RUN cp /usr/lib/postgresql/15/lib/pg_graphql* /opt/bitnami/postgresql/lib/
+RUN cp /usr/share/postgresql/${PG_GRAPHQL_VERSION}/extension/pg_graphql* /opt/bitnami/postgresql/share/extension/
+RUN cp /usr/lib/postgresql/${PG_GRAPHQL_VERSION}/lib/pg_graphql* /opt/bitnami/postgresql/lib/
+
+# Install precompiled supabase wrappers extensions
+RUN curl -L "https://github.com/supabase/wrappers/releases/download/v${WRAPPERS_VERSION}/wrappers-v${WRAPPERS_VERSION}-pg${PG_MAIN_VERSION}-${TARGETARCH}-linux-gnu.deb" -o wrappers.deb
+RUN dpkg --install wrappers.deb
+RUN cp /usr/share/postgresql/${PG_GRAPHQL_VERSION}/extension/wrappers* /opt/bitnami/postgresql/share/extension/
```

Comment:
I'm surprised about PG_GRAPHQL_VERSION being used for wrappers?

Reply:
yes, it is a mistake thanks. Even for `pg_graphql` is it weird isn't it?